util/build_configs.py

```diff
@@ -470,6 +475,11 @@ def _format_text(self, text):
         action='store_true',
         help='Verbose output.'
     )
+    parser.add_option(
+        '-n', '--dryrun',
```

Comment:
Maybe nitpicky: `-n` seems like an odd 1-character flag for `--dryrun` to me. Is this a common flag for dry run?

Reply:
```
$ man make
...
       -n, --just-print, --dry-run, --recon
            Print the commands that would be executed, but do not execute them.
```
----
```
$ git clean --help
...
       -n, --dry-run
           Don't actually remove anything, just show what would be done.
```

How about I change the long name to --dry-run? That's what I meant anyway! 
Thanks.

Reply:
Sounds good to me!